packages/gestalt/src/SideNavigation.css

```diff
@@ -14,13 +15,39 @@
 
 .fullHeight {
   height: 100%;
-  overflow: scroll;
+  overflow-y: scroll;
+}
+
+.contentWidthTransition {
+  transition: width 0.2s;
```

Comment:
Neat. Follow up explorations: can we make consistent the slide in/out transitions across components?

import AnimationProvider from './animation/AnimationContext';